erpnext/manufacturing/doctype/routing/test_routing.py

```diff
@@ -48,7 +46,40 @@ def test_sequence_id(self):
 		wo_doc.cancel()
 		wo_doc.delete()
 
+	def test_update_bom_operation_time(self):
+		operations = [
+			{
+				"operation": "Test Operation A",
+				"workstation": "_Test Workstation A",
+				"hour_rate_rent": 300,
+				"hour_rate_labour": 750 ,
+				"time_in_mins": 30
+			},
+			{
+				"operation": "Test Operation B",
+				"workstation": "_Test Workstation B",
```

Comment:
This test will likely fail if both operations are performed on same workstation.